### ISSUE TITLE
Host not filled in share album button

### ISSUE THREAD

**Issue by vitis586** [petitssuisses/piwigo-ShareAlbum#47]
Hi,

I am not sure what to put into the subject. This plugin was working fine until the 2.10 I think. I do not recall exactly. When I hover over the share album button the browser shows this target URL
_http://piwigo/?xact=activate&cat=1000_ instead of a _http://photos.example.com/?xact=activate&cat=1000_. If I overwrite the URL manually it still works.

I am using Bootstrap Darkroom 2.4.4. The rest of host is

    Piwigo 2.10.2 
    OS: Linux
    PHP: 7.3.16 [2020-04-12 22:57:37]
    MySQL: 5.5.5-10.4.12-MariaDB-1:10.4.12+maria~bionic-log [2020-04-12 22:57:37]
    ImageMagick 7.0.9-7

**Comment by vitis586** [comment]
I tried digging more into this and found a solution - maybe not the best one.

In `public_events.inc.php` I found out that all the buttons are using `get_absolute_root_url()` which only returns the _piwigo_ part for me. That is the name of the docker container, so it might be getting there from the reverse proxy? Not sure.
In `functions_url.inc.php` (part of piwigo base) I found that there is also `get_root_url()` function. I am seeing expected behaviour when I substituted `get_absolute_root_url()` with `get_root_url()`.

Can this be the fix?
